digdag-core/src/main/java/io/digdag/core/agent/ConfigEvalEngine.java

```diff
@@ -203,7 +212,13 @@ private JsonNode evalValue(ObjectNode local, String code)
             for (Map.Entry<String, JsonNode> pair : ImmutableList.copyOf(local.fields())) {
                 scopedParams.set(pair.getKey(), pair.getValue());
             }
-            String resultText = invokeTemplate(templateInvocable, code, scopedParams);
+            String resultText = null;
+            if (requireInvokedTemplate(code)) {
```

Comment:
This is minor but you may reconsider this method name.
boolean methods used in digdag code are https://github.com/treasure-data/digdag/search?q=%22public+boolean%22&unscoped_q=%22public+boolean%22

Reply:
Thank you for your advice. I agree and  I will fix it.

Reply:
Fixed it.